database/mock_data/PopulateDB.sql

```diff
@@ -145,14 +145,13 @@ insert into member (member_type_id, first_name, last_name, phone_number, occupat
 insert into member (uuid, member_type_id, first_name, last_name, phone_number, occupation, email, birthdate, date_joined, active) values ('7b', 1, 'Krispin', 'Deackes', '970-123-8696', 'Analyst Programmer', 'kdeackes2l@imgur.com', '1961-02-14', '2015-11-04', 1);
 insert into member (member_type_id, first_name, last_name, phone_number, occupation, email, birthdate, date_joined, active) values (4, 'Andria', 'Platfoot', '392-414-6161', 'Budget/Accounting Analyst III', 'aplatfoot2m@arizona.edu', '1953-09-28', '2021-03-23', 1);
 insert into member (uuid, member_type_id, first_name, last_name, phone_number, occupation, email, birthdate, date_joined, active) values ('oo7', 4, 'Luelle', 'Bonham', '863-397-4190', 'Desktop Support Technician', 'lbonham2n@thetimes.co.uk', '2002-12-05', '2019-02-10', 1);
-insert into member (uuid, member_type_id, first_name, last_name, phone_number, occupation, email, birthdate, date_joined, active) values ('28m', 3, 'Vasilis', 'Mertel', '561-878-3783', 'Developer II', 'vmertel2o@parallels.com', '1976-11-30', '2019-02-03', 1);
 insert into member (member_type_id, first_name, last_name, phone_number, occupation, email, birthdate, date_joined, active) values (3, 'Kathie', 'O''Scannill', '760-880-8632', 'Tax Accountant', 'koscannill2p@myspace.com', '2009-12-18', '2020-10-15', 1);
 insert into member (member_type_id, first_name, last_name, phone_number, occupation, email, birthdate, date_joined, active) values (2, 'Patin', 'Lelliott', '913-810-0953', 'Senior Financial Analyst', 'plelliott2q@multiply.com', '2005-12-11', '2018-05-19', 1);
+insert into member (uuid, member_type_id, first_name, last_name, phone_number, occupation, email, birthdate, date_joined, active) values ('28m', 3, 'Vasilis', 'Mertel', '561-878-3783', 'Developer II', 'vmertel2o@parallels.com', '1976-11-30', '2019-02-03', 1);
```

Comment:
I think this is the reason why the integration test is failing. Why did you do this?

Reply:
Honestly... no idea